awesomplete.css

```diff
@@ -1,4 +1,14 @@
 [hidden] { display: none; }
+.visually-hidden {
+	border: 0;
+	clip: rect(0 0 0 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+}
```

Comment:
You are right. For this use case some of the properties are not needed. I updated the CSS accordingly.